agents/sources.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+const opentrialsApi = require('../config').opentrialsApi;
+
+function listSources() {
+  return opentrialsApi
+    .then((client) => client.sources.list())
+    .then((response) => response.obj);
+}
+
+module.exports = {
+  list: listSources,
```

Comment:
Depends on what get merged first, if this or https://github.com/opentrials/api/pull/80. As this is an easier PR, I would prefer to merge it before and then do the fixes as part of other PR, but we'll see.